src/plugins/exception-helpers.ts

```diff
@@ -14,32 +17,32 @@ export default function exceptionHelpersPlugin (app) {
   app.addHelper('websocketException', websocketException);
 }
 
-function exception (ctx, error) {
+async function exception (ctx: MojoDualContext, error: Error): Promise<boolean> {
   if (ctx.isWebSocket) return ctx.websocketException(error);
-  return ctx.httpException(error);
+  return await httpException(ctx, error);
```

Comment:
This no longer uses the helper. Break being able to change behaviour with new helpers.